frontend/src/soundEffect.ts

```diff
@@ -0,0 +1,19 @@
+const soundEffects = {

```

Comment:
Consider making this an enum rather than an object.

Reply:
Can typescript enums be indexed with variables?

Reply:
I guess so! It's kind of weird how JS objects are both hashmaps/dicts and JS' way of doing inheritence.
![image](https://github.com/user-attachments/assets/2ed04da4-6879-47e2-9d77-9a1189de8e24)